packages/ocaml-variants/ocaml-variants.5.1.0~alpha2+options/opam

```diff
@@ -0,0 +1,86 @@
+opam-version: "2.0"
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+synopsis: "Second alpha release of OCaml 5.1.0"
+maintainer: "platform@lists.ocaml.org"
+authors: ["Xavier Leroy" "Damien Doligez" "Alain Frisch" "Jacques Garrigue" "Didier Rémy" "Jérôme Vouillon"]
+homepage: "https://ocaml.org"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "git+https://github.com/ocaml/ocaml.git#5.1"
+depends: [
+  "ocaml" {= "5.1.0" & post}
+  "base-unix" {post}
+  "base-bigarray" {post}
+  "base-threads" {post}
+  "base-domains" {post}
+  "base-nnp" {post}
+  "ocaml-option-bytecode-only" {arch != "arm64" & arch != "x86_64"}
```

Comment:
```suggestion
  "ocaml-option-bytecode-only" {arch != "arm64" & arch != "x86_64"}
```

I'm not sure what the reasoning behind this restriction is. It should be possible to compile ocaml as bytecode only on arm64 or x86_64 (it just passes --disable-native-compiler).  But if we _do_ want to keep this restriction, then the architecture set should probably include riscv and s390x now in 5.1.0alpha2.